src/versions/validation/event.js

```diff
@@ -10,10 +10,56 @@ const orderByFields = [
 
 const schemas = {}
 
+const getObjectSchema = (name) => Joi.string().regex(new RegExp(`^${name}\\.\\w+$`))
```

Comment:
It is impossible to use nested properties with this regexp. Is it what you expect, and why?

Reply:
Good catch. I'll update the regexp so deep nested properties can be used.